src/index.html

```diff
@@ -11,8 +11,60 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
```

Comment:
You should link a CSS file instead of SCSS. Browsers do not understand SCSS syntax. Your SCSS should be compiled into CSS and then linked in your HTML.